go.mod

```diff
@@ -30,17 +30,17 @@ require (
 	github.com/tevino/abool v1.2.0
 	github.com/yanniszark/go-nodetool v0.0.0-20191206125106-cd8f91fa16be
 	gopkg.in/ini.v1 v1.57.0
-	k8s.io/api v0.20.0
-	k8s.io/apiextensions-apiserver v0.20.0
-	k8s.io/apimachinery v0.20.0
+	k8s.io/api v0.20.1
+	k8s.io/apiextensions-apiserver v0.20.1
+	k8s.io/apimachinery v0.20.1
 	k8s.io/apiserver v0.20.0
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/cloud-provider v0.20.0
 	k8s.io/code-generator v0.20.0
 	k8s.io/kube-controller-manager v0.20.0
 	k8s.io/kubernetes v1.20.0
```

Comment:
k8s conversion to v1.20.1?

Reply:
Yes, it's being pulled because controller-runtime 0.8 supports 1.20.1 I believe.